spec/controllers/profile_controller_spec.rb

```diff
@@ -12,7 +12,8 @@
     it 'should create a new profile if one does not exist' do
       @user = FactoryGirl.create(:user)
       @user.profile.destroy
-      get :show, user: @user.id
+      @user.reload
+      get :show, user_id: @user
```

Comment:
I am aware we are just about to remove the bit that creates a new preference if the user doesn't have one.

https://github.com/otwcode/otwarchive/pull/2668/files#diff-2a02e520fe278dced5dea79dbb1c0ef4R14

What should we do instead ?